installation/docker/apiserver.env

```diff
@@ -0,0 +1,8 @@
+# DATABASE_DRIVER=POSTGRES
+# DB_NAME=vmclarity
+# DB_USER=vmclarity
+# DB_PASS=__PostgresDBPassword__
+# DB_HOST=postgres.service
+# DB_PORT_NUMBER=5432
+DATABASE_DRIVER=LOCAL
```

Comment:
we don't want postgres by default?

Reply:
postgres requires some extra configuration from the user such as configuring the password etc. SQLite works out of the box and docker volumes give it persistence for free.